### PR TITLE
Fesom2Geoformats

### PR DESCRIPTION
**Pull request by alirezamdv**
This function is responsible for creating the Gdal datasets(GeoTiff, NetCDF) from the Fesom result For more information please take a look at the notebook example.

### CODE REVIEW

File: notebooks/Fesom2GeoFormat.ipynb [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    "### Set Path to the mesh and result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "meshPath  ='tests/data/pi-grid/'\n",
```
Comment (koldunovn) [inline notebooks/Fesom2GeoFormat.ipynb:43]: It should be changed to "../tests/data/pi-grid/" to make it work in the `notebooks` folder.

File: notebooks/Fesom2GeoFormat.ipynb [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    "result_path ='tests/data/pi-results/temp.fesom.1948.nc'\n",
```
Comment (koldunovn) [inline notebooks/Fesom2GeoFormat.ipynb:44]: Same here "../tests/data/pi-grid/"

File: pyfesom2/fesom2GeoFormat.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
+from pyfesom2 import fesom2regular
+from osgeo import osr, gdal
```
Comment (koldunovn) [inline pyfesom2/fesom2GeoFormat.py:2]: I would make the osgeo dependency optional. Can you, please, change to something like:
```python
try:
    from osgeo import osr, gdal
except ImportError:
    print('osgeo is not installed, conversion to vector format (fesom2GeoFormat) will not work.')

File: pyfesom2/fesom2GeoFormat.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+import numpy as np
+
+
+def fesom2GeoFormat(data, mesh, outName, radius_of_influence=100000, methode='nn', driver='Gtiff'):
```
Comment (koldunovn) [inline pyfesom2/fesom2GeoFormat.py:6]: can you, please, rename `methode` to `method`. Let's stick to English :)

File: pyfesom2/fesom2GeoFormat.py [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    """ Returns Fesom as Gdal Geotiff or Gdal NetCDF .
+
+    Parameters:
+        data (array): which is come from select_slices().
+        mesh (fesom mesh): which is come from load_mesh().
+        outName (str): output Name.
+        methode : interpolation methode for Fesom2Regular().
```
Comment (koldunovn) [inline pyfesom2/fesom2GeoFormat.py:13]: please rename

File: pyfesom2/fesom2GeoFormat.py [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    outputName = outName
+    array = fesom2regular(
+        data2,
+        mesh,
+        lonreg,
+        latreg,
+        distances_path=None,
+        inds_path=None,
+        qhull_path=None,
+        how=methode,
```
Comment (koldunovn) [inline pyfesom2/fesom2GeoFormat.py:33]: plaese rename

Comment (alirezamdv) [reply pyfesom2/fesom2GeoFormat.py:33]: of course :), I am going to change it, thanks for your review.

**Comment by alirezamdv** [comment]
Thanks Nikolay, have a nice weekend.

**Pull request merged**